### DISCUSSION TITLE
Mk3s+ + Pi 3B+ and klipperscreen

### DISCUSSION THREAD

**Discussion by kaisarantonn** [dz0ny/klipper-prusa-mk3s#4] [Q&A] [answered]
i added printer.cfg form your config at mainsailos.local and this happend

An error was detected while loading the moonraker components. Please check the logfile and fix the issue.

Following plugin has an error:

update_manager
Unparsed config option 'origin: https://github.com/dz0ny/klipper-prusa-mk3s.git' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'primary_branch: main' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'is_system_service: False' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

**Comment by dz0ny** [comment]
Sounds like you have very old version of **moonraker**, if you visit /config you will have option there to update it.

**Reply by kaisarantonn** [reply]
> Sounds like you have very old version of **moonraker**, if you visit /config you will have option there to update it.

no, updated the latest moonraker

in the machine page it says

Include file '/home/pi/klipper_config/config/mk3s/eisny-rambo.cfg' does not exist

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted

**Comment by dz0ny** [comment]
This is after update :) In printer.cfg update **eisny-rambo.cfg** to **einsy-rambo.cfg**.

**Comment by kaisarantonn** [comment]
nope, not fixing anything it is kind of not finding your repo

Moonraker Warnings

An error was detected while loading the moonraker components. Please check the logfile and fix the issue.

Following plugin has an error:

update_manager
Unparsed config option 'ype: git_repo' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'origin: https://github.com/dz0ny/klipper-prusa-mk3s.git' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'path: ~/klipper_config/config' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'primary_branch: main' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

Unparsed config option 'is_system_service: False' detected in section [update_manager prusa]. This may be an option no longer available or could be the result of a module that failed to load. In the future this will result in a startup error.

**Comment by dz0ny** [comment]
You need to remove the above config, restart the service, update moonraker, then add it back.  Note that updates won't be listed unless you click on refresh button.

**Reply by rubino25** [reply]
Hi! I have the same issue!

**Reply by dz0ny** [reply]
Did you do step 3? https://github.com/dz0ny/klipper-prusa-mk3s#install:~:text=Update%20all%20components%20under%20Machine%20tab%2C%20otherwise%20config%20might%20not%20be%20able%20to%20load errors are related to outdated components.

**Reply by syixxs** [reply]
Hey dz0ny, running into the same error. As reported by the Mainsail notification: 
![image](https://user-images.githubusercontent.com/7157306/174726790-996e8140-a817-4369-9d58-54a2c890baf9.png)

My version of Moonraker and the other components: 
![image](https://user-images.githubusercontent.com/7157306/174726873-d6e9106d-b591-4168-bcff-800bd1cd405d.png)

I've cleared the update manager piece from the moonraker.conf, rebooted the Pi, came up without the errors. Upon adding the update manager entry again, however, same errors. As far as I can tell that's what you've been asking folks to do. There aren't any updates available to install and refreshing between removing the stanza and adding it doesn't do the trick.

I'm at a loss at this point, as it appears that all the information in the stanza is correct and accurate as far as the Moonraker docs are concerned. I even attempted to add the ~/klipper_config/config/ directory manually since the first error in the list mentions the path doesn't exist. That didn't do the trick, either.

Help, please!

**Reply by syixxs** [reply]
Lacking Moonraker's repo functionality, I manually cloned your repo and distributed the files - that got everything working. The issue is clearly Moonraker related, as your settings and config files are doing the job brilliantly. Just not sure what's up with Moonraker failing to parse the update manager config you've provided.

**Comment by dz0ny** [comment, marked as answer]
I've updated the docs, thanks for reporting
